account_operating_unit/views/account_journal_view.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!--© 2016-17 ForgeFlow S.L.-->
+<!--© 2016 Serpent Consulting Services Pvt. Ltd.-->
+<!--License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).-->
+<odoo>
+    <record id="view_account_journal_form" model="ir.ui.view">
+        <field name="name">account.journal.form</field>
+        <field name="model">account.journal</field>
+        <field name="inherit_id" ref="account.view_account_journal_form" />
+        <field name="arch" type="xml">
+            <field name="company_id" position="after">
+                <field
```

Comment:
I cannot see this field in journals in runbot. Can you check?

Reply:
It's the normal company_id field.

Reply:
with this fix: 
![image](https://github.com/OCA/operating-unit/assets/2032807/7011a190-ba95-40f3-990f-ca8bfa304f86)